### PR TITLE
check timestamp from group device

### PR DESCRIPTION
**Pull request by AlvaroVega**
Implements issue https://github.com/telefonicaid/iotagent-node-lib/issues/655
- [x] upate documentation
- [x] upate/add some specific test

(To be merged with squash)

### CODE REVIEW

**Comment by AlvaroVega** [comment]
To be merged with squash

File: README.md [hunk 1]
```diff
@@ -160,10 +160,10 @@ requested action. Parameters for the command will be passed inside the command v
...
 * First of all, for every command defined in a device, two new attributes are created in the entity with the same name
 as the command but with a prefix:
-	* `_info`: this attribute reflect the current execution status of the command. When a command request is issued by
-	the Context Broker, the IoT Agent library generates this attribute with 'PENDING' value. The value of this attribute
-	will be changed each time a command error or result is issued to the IoT Agent.
-	* `_result`: this attribute reflect the result of the execution of the defined command.
+        * `_info`: this attribute reflect the current execution status of the command. When a command request is issued by
+        the Context Broker, the IoT Agent library generates this attribute with 'PENDING' value. The value of this attribute
+        will be changed each time a command error or result is issued to the IoT Agent.
+        * `_result`: this attribute reflect the result of the execution of the defined command.
```
Comment (fgalan) [inline README.md]: The changes in this part seems to have  broken the render:

imagen

File: README.md [hunk 2]
```diff
@@ -260,7 +260,7 @@ once every day, at midnight, the `TimeInstant` reported for each measurement wil
 such measurements, the `TimeInstant` attribute would take those values around midnight.
 
 This functionality can be turned on and off through the use of the `timestamp` configuration flag (described in the
-configuration).
+configuration), as well ass 'timestamp' flag in device or group provision.
```
Comment (fgalan) [inline README.md]: ass -> as

File: doc/installationguide.md [hunk 3]
```diff
@@ -10,10 +10,10 @@ These are the parameters that can be configured in the global section:
 * **contextBroker**: connection data to the Context Broker (host and port). E.g.:
```
Comment (fgalan) [inline doc/installationguide.md]: CNR entry missing

**Comment by fgalan** [comment]
Only some minor comments.

@dcalvoalonso could you also have a look and provide LGTM if you find it ok (apart from the comments already raised by me), pls? Thx!

File: doc/installationguide.md (hunk 3 above)
Comment (AlvaroVega) [reply doc/installationguide.md]: fixed in d7a481626c1703026f93509bf784c4f24985f866

File: README.md (hunk 2 above)
Comment (AlvaroVega) [reply README.md]: fixed in 5d2d3f3cfa54d855bef864cd2efd8c8c52d7eb85

File: CHANGES_NEXT_RELEASE [hunk 4]
```diff
@@ -1,5 +1,6 @@
+Add trimestamp in device and group provision (#655)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: trimestamp -> timestamp

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE]: fixed in e47193061f69165511c7d854f1832d93730b1b24

File: doc/installationguide.md [hunk 5]
```diff
@@ -136,7 +136,7 @@ are 5 and 5 respectively). E.g.:
 have implications in the use of attributes with Context Providers, so this flag should be used with care.
 * **dieOnUnexpectedError**: if this flag is activated, the IoTAgent will not capture global exception, thus dying upon any unexpected error.
 * **singleConfigurationMode**: enables the Single Configuration mode for backwards compatibility (see description in the Overview). Default to false.
-* **timestamp**: if this flag is activated, the IoT Agent will add a 'TimeInstant' metadata attribute to all the attributes updateded from device information.
+* **timestamp**: if this flag is activated, the IoT Agent will add a 'TimeInstant' metadata attribute to all the attributes updated from device information. This flag is overwrited `timestamp` flag in group or device provision.
```
Comment (fgalan) [inline doc/installationguide.md]: overwrited -> overwritten by ?

File: README.md (hunk 1 above)
Comment (AlvaroVega) [reply README.md]: Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/669/commits/e43c1d54fd2e857b40c6751488addc3e1462bde2

File: doc/installationguide.md (hunk 5 above)
Comment (AlvaroVega) [reply doc/installationguide.md]: fixed by https://github.com/telefonicaid/iotagent-node-lib/pull/669/files#diff-9b0a554150d1aeae259b560098da86f1

**Comment by dcalvoalonso** [comment]
Sorry for the delay providing feedback @fgalan @AlvaroVega , I missed somehow the notification from GitHub.

LGTM! No comments from my side.

**Pull request merged**